lib/translations_checker/checker.rb

```diff
@@ -70,12 +72,20 @@ def translated_locales
       @translated_locales ||= locales - %w(en en-AU)
     end
 
+    def git_diffs
+      GitDiff.call(LOCALES_DIR)
+    end
+
     def diffs
-      @diffs ||= GitDiff.call(LOCALES_DIR).map do |path, hunks|
+      @diffs ||= supported_git_diffs.map do |path, hunks|
         Diff.new(path, hunks)
       end
     end
 
+    def supported_git_diffs
+      git_diffs.select { |path, hunks| SUPPORTED_FORMATS.include?(path.split('.').last) }
```

Comment:
I think it would be better to use `File.extname` to extract the file extension. This will mean adding `.` to each entry in `SUPPORTED_FORMATS`. Also, if you're not using a parameter in a method or block, it's common practice to use `_` as a placeholder parameter name.
```suggestion
      git_diffs.select { |path, _| SUPPORTED_FORMATS.include?(File.extname(path)) }
```